x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/components/steps/step_define_package_policy.tsx

```diff
@@ -44,7 +44,7 @@ import { PackagePolicyInputVarField } from './components';
 // on smaller screens, fields should be displayed in one column
 const FormGroupResponsiveFields = styled(EuiDescribedFormGroup)`
   @media (max-width: 767px) {
-    .euiFlexGroup--responsive {
+    [class*='euiFlexGroup-responsive'] {
```

Comment:
is this a change in styled components or only an alternative way of defining class?

Reply:
An alternate way of defining class. This selector change is due to EUI's output classes changing from our Emotion conversions. We're generally removing modifier classes (but keeping parent classNames like `.euiFlexGroup`), but technically those modifiers are still accessible via Emotion's generated class name concatenation:

<img width="816" alt="" src="https://user-images.githubusercontent.com/549407/201993916-38df8f2b-5b93-4ac5-a655-00f7eeff01b3.png">

This change is essentially a quick fix to maintain current working behavior - without it, your selector would no longer be working after this upgrade. In general, or for a more long-term fix, we recommend passing in your own custom classNames (conditionally if necessary), and recommend against targeting our state-based classNames (our classNames have never been a guaranteed API/hook).